guides/doc-Installing_Server_foremanctl/docinfo.xml

```diff
@@ -0,0 +1,15 @@
+<title>{InstallingServerDocTitle}</title>
```

Comment:
I'm wondering if this guide should get a more specific title, both upstream and downstream?
Considering they will be published side by side, it would be confusing to have two guides with the same title.

Reply:
Good point, how about we just add "by using foremanctl" for now? I think that should be good enough -- we're just setting up a very simple preview guide that we're not even adding to the navigation (yet), no need to come up with a definitive title at this stage.